jwql/utils/utils.py

```diff
@@ -275,3 +370,30 @@ def get_config():
         settings = json.load(config_file)
 
     return settings
+
+
+def filesystem_path(filename):
+    """Return the full path to a given file in the filesystem
+
+    Parameters
+    ----------
+    filename : str
+        File to locate (e.g. jw86600006001_02101_00008_guider1_cal.fits)
+
+    Returns
+    -------
+    full_path : str
+        Full path to the given file, including filename
+    """
+    filesystem_base = get_config()["filesystem"]
+
+    # Subdirectory name is based on the proposal ID
+    subdir = filename[0:7]
```

Comment:
@bhilbert4 I would suggest utilizing `filename_parser` for this, in case for some reason these indices change one day.

Reply:
Done